connectors/spark/src/main/scala/io/unitycatalog/spark/RetryableTemporaryCredentialsApi.java

```diff
@@ -0,0 +1,178 @@
+package io.unitycatalog.spark;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Throwables;
```

Comment:
Please be careful to import the 3rd-party deps such as `guava`, since for different guava , it does not provide the sufficient compatibility. 

For the `unitycatalog-spark-connector` usually we use the `import org.sparkproject.guava` . Please change to use that one, otherwise it may conflict with the runtime's deps.